packages/kbn-ftr-common-functional-services/services/bsearch.ts

```diff
@@ -70,14 +70,15 @@ export class BsearchService extends FtrService {
   /** Send method to send in your supertest, url, options, and strategy name */
   async send<T extends IEsSearchResponse>({ supertest, options, strategy, space }: SendOptions) {
     const spaceUrl = getSpaceUrlPrefix(space);
-    const { body } = await this.retry.try(async () => {
+    const makeRequest = async () => {
       return supertest
         .post(`${spaceUrl}/internal/search/${strategy}`)
         .set(ELASTIC_HTTP_VERSION_HEADER, '1')
         .set('kbn-xsrf', 'true')
-        .send(options)
-        .expect(200);
```

Comment:
Could you please explain why you've dropped the `.expect(200)`? From what I can tell, this was the bit that triggered the retry in case it was failing. Without it, I think the code would just move on no matter of the request result.

Reply:
Thanks catching it. This was an oversight on my part while debugging. I have adding it back again.

Reply:
This is done as well. Thanks.